lib/node_modules/@stdlib/datasets/penguins/docs/repl.txt

```diff
@@ -0,0 +1,70 @@
+
+{{alias}}()
+    Returns the Palmer Penguins dataset as an alternative to the Iris Dataset.
+
+    The data consists of 17 attributes:
+
+    - studyName: Sampling expedition from which data were collected.
+    - sampleNumber: An integer denoting the continuous numbering sequence for 
+    each sample.
+    - species: A string denoting whether the observation belongs to an Adelie, 
+    Chinstrap or a Gentoo Penguin.
+    - region: A string denoting the region of Palmer LTER sampling grid.
+    - island: Island name (Dream, Torgersen, or Biscoe) in the Palmer 
+    Archipelago (Antarctica).
+    - stage: A string denoting reproductive stage at sampling.
+    - individualID: A string denoting the unique ID for each individual in 
+    dataset.
+    - clutchCompletion: A boolean whether the study nest was observed with a 
+    full clutch, i.e., 2 eggs.
+    - dateEgg: The date when the study nest was observed with 1 egg.
+    - culmenLength: The length of the dorsal ridge of a bird's bill 
+    (millimeters).
+    - culmenDepth: The depth of the dorsal ridge of a bird's bill 
+    (millimeters).
+    - flipperLength: The length penguin flipper (millimeters).
+    - bodyMass: The penguin's body mass in grams.
+    - sex: The sex of the penguin.
+    - delta15N: The measure of the ratio of stable isotopes 15N:14N.
+    - delta13C: The measure of the ratio of stable isotopes 13C:12C.
+    - comments: Additional relevant information for the observation.
+
+    The palmerpenguins data contains size measurements for three penguin 
+    species observed on three islands in the Palmer Archipelago, Antarctica.
+    The dataset can be used to classify two dependent variables: 1) sex 
+    and 2) species.
+
+    Returns
+    -------
+    out: Array<Object>
+        Palmer peguins data.
+
+    Examples
+    --------
+    > var data = {{alias}}()
+    [ {...}, {...}, ... ]
+
+    References
+    ----------
+    - Gorman KB, Williams TD, Fraser WR (2014). Ecological sexual dimorphism 
```

Comment:
The references should also be listed in a references section of the `README.md` file. See e.g. [here](https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/datasets/berndt-cps-wages-1985/README.md#references) for an example.

Reply:
Sure I will have this one added